string got_object_action
string got_object_name
float32[] got_euler_pose